dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/MaintenanceController.java

```diff
@@ -68,6 +70,7 @@
 @OpenApi.Tags("system")
 @Controller
 @RequestMapping(value = MaintenanceController.RESOURCE_PATH)
+@RequiresAuthority(anyOf = F_PERFORM_MAINTENANCE)
```

Comment:
Added check at class level as only 2 methods don't require this authority and they each have their own checks.

Reply:
I know, I thought the same myself. Repeating something 21 times when 1 will do seems a bit unreasonable.
I can change it back, let me know.

Reply:
Lets discuss it in tomorrows meeting

Reply:
we agreed at the team meeting to allow class & method level